src/components/common/DropdownKebab/DropdownKebab.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from 'react';
+import more from '../../../images/icons/more.svg';
+import editImage from '../../../images/icons/edit.svg';
+import styles from './DropdownKebab.module.css';
+
+export default function DropdownKebab({ List, handleButtonClick }) {
```

Comment:
```suggestion
export default function DropdownKebab({ list, handleButtonClick }) {
```
대문자로 시작하는 부분은 대부분 컴포넌트의 이름이기 때문에 이 부분은 소문자로 작성해 주세요~

Reply:
넵! 소문자로 수정하였습니다!